sphinxcontrib/confluencebuilder/config.py

```diff
@@ -123,4 +123,36 @@ def validate(c, log=True):
 properly set with the publisher's Confluence username.
 """)
 
+            if c.confluence_ca_cert:
+                if (not os.path.isfile(c.confluence_ca_cert) and
```

Comment:
Can't we just use `os.path.exists` instead? (I could be wrong)

Reply:
Yay! Now it fits on one line :)